src/main/resources/templates/fragments/attribution.html

```diff
@@ -3,7 +3,6 @@
   <body>
   <th:block th:fragment="attribution">
     <div class="organisation">
-      <h2 class="heading-small">Managed by</h2>
```

Comment:
Can we delete this file as I'm not sure it's being used anymore?

Reply:
Yes, agreed. I need your help